python/test/t_LOLAVoronoi_std.py

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+import openturns.testing as ott
+
+ot.TESTPREAMBLE()
+
+f1 = ot.SymbolicFunction(
+    ["a0", "a1"],
+    ["-4 * exp((-25 / 8) * (a0^2 + a1^2)) + 7 * exp((-125 / 4) * (a0^2 + a1^2))"] * 2
+)
+
+distribution = ot.JointDistribution([ot.Uniform(-1.0, 1.0)] * 2)
+x0 = ot.LowDiscrepancyExperiment(ot.HaltonSequence(), distribution, 100).generate()
+y0 = f1(x0)
+
+# reference mean for each generated batch of input x
+ref_mean = [
+    [-0.00975428, 0.0114831],
+    [0.157271, 0.117598],
+    [0.00837916, 0.00994911],
+    [0.11306, -0.085921],
+    [0.0737508, 0.110677],
+]
+
+algo = otexp.LOLAVoronoi(x0, y0, distribution)
```

Comment:
There are two flavors of LOLAVoronoi depending on NonLinearityAggregationMethod:
- Maximum
- Average

Please make a loop over these options and check that the code does not fail.

There are also two flavors of LOLAVoronoi depending on the `MaximumCombinationsNumber`. Please set it to 0 and check that the "else" branch succeeds. It is currently not covered according to CodeCov.

![image](https://github.com/user-attachments/assets/1f9ed21b-9783-4a23-b97e-63ba6911ab3d)